packages/xo-web/src/xo-app/vm/tab-advanced.js

```diff
@@ -833,6 +841,22 @@ export default class TabAdvanced extends Component {
                     </td>
                   </tr>
                 )}
+                <tr>
+                  <th>{_('vmBootFirmware')}</th>
+                  <td>
+                    <select
+                      className='form-control'
+                      onChange={this._onBootFirmwareChange}
+                      value={defined(() => vm.boot.firmware, 'bios')}
+                    >
+                      {VM_BOOT_FIRMWARES.map(val => (
+                        <option key={val} value={val}>
+                          {val}
+                        </option>
+                      ))}
+                    </select>
+                  </td>
+                </tr>
```

Comment:
I think it should only be displayed for `HVM` VMs, and probably only those on CH/XCP-ng >= 8.0.

/cc @olivierlambert Do you agree?

Reply:
Indeed, UEFI doesn't work in PV guests and only in CH/XCP-ng 8.0. However, should we gray out the option or hide it? Graying with a tooltip might be more informative I think.

Reply:
should we display it also for `pvhvm` ?

Reply:
There's only 2 virt modes in Xen world: PV or HVM. PVHVM is a kind of "subset" of HVM.